src/main/scala/com/gu/support/workers/lambdas/CreateSalesforceContact.scala

```diff
@@ -16,13 +18,18 @@ class CreateSalesforceContact extends ServicesHandler[CreateSalesforceContactSta
 
   override protected def servicesHandler(state: CreateSalesforceContactState, requestInfo: RequestInfo, context: Context, services: Services) = {
     SafeLogger.debug(s"CreateSalesforceContact state: $state")
+
+    val addressLine = combinedAddressLine(state.user.billingAddress.lineOne, state.user.billingAddress.lineTwo) map asFormattedString
```

Comment:
I think everywhere else in the codebase we would write `.map(asFormattedString)` might be worth keeping it consistent

Reply:
Sure. Will do.